packages/app/src/debug/DebugCancelledAlert.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <Alert
+    :title="t('debugPage.manuallyCancelled')"
+    status="warning"
+    :icon="ErrorOutlineIcon"
+    class="flex flex-col mb-24px w-full"
+  >
+    <div class="flex items-center">
```

Comment:
Styling of this feels very squished, is there meant to be space between these items?
![Screen Shot 2022-12-28 at 1 31 17 PM](https://user-images.githubusercontent.com/11482842/209863128-837812a4-a5e0-4e7f-af53-8a6fb7f57004.png)

Reply:
yeah, it's supposed to look like this

![Screen Shot 2022-12-28 at 3 17 52 PM](https://user-images.githubusercontent.com/14275198/209867528-0ecf3a33-afa2-4858-b9f1-7d6627a7c9d8.png)

where are you seeing that? ^

Reply:
Hmm, was seeing it in the `DebugRunStates.cy.tsx` tests but looks good now. Must have been cached/borked local state, disregard.